todoapp-main/src/main/java/ru/todoapp/model/SaveTaskEntity.java

```diff
@@ -0,0 +1,23 @@
+package ru.todoapp.model;
+
+import ru.todoapp.model.dto.AddTaskRequestDTO;
+
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+/**
+ * Task entity to be saved in database
+ *
+ * @param description - task description
+ * @param userUUID    - userUUID of user that planned task
+ * @param datetime    - timestamp then the task planned to take place
+ */
+public record SaveTaskEntity(String description, String userUUID, OffsetDateTime datetime) {
```

Comment:
You don't need two classes to represent one entity. Instead of this, you can create one class with the same fields as in the corresponding table. After that, you just fill the only needed fields. For example, when you create entity before saving it, you just don't fill the ID field ('cause you don't know it value yet)

Reply:
Ok